synapse/storage/databases/main/search.py

```diff
@@ -33,13 +33,19 @@
 
 
 class SearchWorkerStore(SQLBaseStore):
-    def store_search_entries_txn(self, txn, entries):
+    def _clean_value_for_search(self, value: str) -> str:
+        """Replaces null code points in string as postgres/sql do not
+        like the insertion of strings with null code points"""
+        return value.replace("\u0000", " ")
```

Comment:
I think we can move this outside the class since it isn't really related to the store (we don't need `self`).